src/main/resources/wrong-secrets-configuration.yaml

```
@@ -766,3 +766,17 @@ configurations:
       ctf:
         enabled: true
       missing_environment: "explanations/missing_vault.adoc"
+
+    - name: Challenge 48
+        short-name: "challenge-48"
+        sources:
+          - class-name: "org.owasp.wrongsecrets.challenges.kubernetes.Challenge48"
+            explanation: "explanations/challenge48.adoc"
+            hint: "explanations/challenge48_hint.adoc"
+            reason: "explanations/challenge48_reason.adoc"
+            environments: [ *k8s, *gcp, *aws, *azure ]
+        difficulty: *normal
+        category: *secrets
+        ctf:
+          enabled: false
+      missing_environment: "explanations/missing_k8s.adoc"
```

```suggestion
      short-name: "challenge-48"
      sources:
        - class-name: "org.owasp.wrongsecrets.challenges.kubernetes.Challenge48"
          explanation: "explanations/challenge48.adoc"
          hint: "explanations/challenge48_hint.adoc"
          reason: "explanations/challenge48_reason.adoc"
          environments: [ *k8s, *gcp, *aws, *azure ]
      difficulty: *normal
      category: *secrets
      ctf:
        enabled: false
    missing_environment: "explanations/missing_k8s.adoc"
```
it looks like it is failing due to identation: can you make `short-name` and the rest allign with `name` place?
and then, with docker for desktop enabled, use `./mvnw clean install` to get it running again.

---

I have created a container with tag  `challenge48-no-vault`  based on the current challenge-code.

---

in order to use it, can you create teh following modifications to your k8s deployment yamls? in https://github.com/Shubham-Patel07/wrongsecrets/blob/fix/Issue858/k8s/secret-challenge-vault-deployment.yml#L53 use `- image: jeroenwillemsen/wrongsecrets:challenge48-k8s-vault` and in https://github.com/Shubham-Patel07/wrongsecrets/blob/fix/Issue858/k8s/secret-challenge-deployment.yml#L31 use `- image: jeroenwillemsen/wrongsecrets:challenge48-no-vault` this should load up the container with the new challenge

---

after this, you will have to updat the k8s deployment manifest to use the sealed secret under the env-var `SEALED_SECRET_ANSWER`

---

hi @commjoen,
I tried using the `challenge48-no-vault` and  `challenge48-k8s-vault` but still the challenge is not available please can you check containers again